src/colornames.csv

```diff
@@ -6133,6 +6147,7 @@ Connaisseur,#654e44,
 Connect Red,#eb6651,
 Connected Grey,#898473,
 Connecticut Lilac,#ccbbee,
+Connoisseur,#6b574e,

```

Comment:
I don't think this conveys a clear color. Lets remove it or come up with an other name